monkey/monkey_island/cc/services/run_local_monkey.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import os
+import platform
+import stat
+import subprocess
+from shutil import copyfile
+
+import monkey_island.cc.environment.environment_singleton as env_singleton
+from monkey_island.cc.resources.monkey_download import get_monkey_executable
+from monkey_island.cc.server_utils.consts import MONKEY_ISLAND_ABS_PATH
+from monkey_island.cc.services.utils.network_utils import local_ip_addresses
+
+logger = logging.getLogger(__name__)
+
+
+class RunLocalMonkeyService:
```

Comment:
Should be a noun like `LocalMonkeyRunService`, now it sounds like it's a method that runs the service.